main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,260 @@
+another, and one which is used by that same object. For each entry,
+the name is linked to its primary page in this Guide. 
+
+- **[Amount](./Amounts.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#amount)**:
+  The description of an asset, it is not an asset itself, having no economic scarcity or economic value.
+  Made up of a:
+  - *[value](./Amounts.md)* *[(Glossary)](https://agoric.com/documentation/glossary/#value)*:  The
+  size of the amount.
+  - *[brand](./Amounts.md)* *[(Glossary)](https://agoric.com/documentation/glossary/#brand)* The
+  kind of asset. 
+  
+  You can think of these as the "how many" and the "what" of an asset, for example, 7 quatloos (an imaginary
+  currency).
+- **[AmountMath](./AmountMath.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#amountmath)**:
+Methods to do math operations on an `amount`..
+- **[Purse](./PursesAndPayments.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#purse)**: an
+  object for long term digital assets storage. 
+- **[Payment](./PuresAndPayments.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#payment)**:
+  an object for transfering digital assets to another party.
+- **[Mint](./IssuersAndMints.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#mint)**: an
+  object that creates new `payment` objects of a specific `brand`. Each `brand` has
+  a one to one relationship with a `mint` and vice versa.
+- **[Issuer](./IssuersAndMints.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#issuer)**: 
+an object linked to a single `mint` object that creates empty `purse` objects
+for that mint's `brand` and operates on any `payment` of that `brand`. 
+
+## Fungible and Non-Fungible Assets
+
+There are two kinds of assets,
+*[fungible](https://agoric.com/documentation/glossary/#fungible)* and
+*[non-fungible](https://agoric.com/documentation/glossary/#non-fungible)*. 
+
+Fungible assets are interchangeable. For example, if you have 100
+one-dollar bills and need to pay someone 5 dollars, it doesn't matter
+which five of your one-dollar bills you give them. 
+
+Non-fungible assets are of the same type, but are not interchangeable
+and specific items must be used. For example, you might have 100
+tickets. But someone wanting to buy even a General Admission ticket from you will want one
+for a specific date and time. This might also affect the price; you'll want to charge more
+for a Friday evening ticket than a Wednesday matinee ticket, even if it's for the same show.
+
+## Amounts are not assets
+
+**IMPORTANT**: Despite how it may seem, an `amount` is only a description of an asset, not
+an asset in and of itself. Actual assets are in `purse` and `payment` objects, but the
+assets aren't ever embodied in any particular JavaScript object. Instead, what makes the 
+assets exist is the issuer having a mapping from the `purse` or `payment` to an `amount` (description).
```

Comment:
I'd like to get Kate's opinion on this. She and I had some back and forth on how I had the mistaken belief that there were amount objects effectively stored in the p/p objects, and I think your phrasing is more likely to cause people to get that belief than an explicit "No, that's not how it works".

Reply:
How about "assets are held by purses and payments in the same way that a bank account holds money - it does so not because of anything physical, but because of a record held by an authoritative source, in this case, the issuer is the source of truth of what assets are held by what purses and payments"